foundry.toml

```diff
@@ -1,4 +1,4 @@
 [profile.default]
-solc_version = "0.8.22"
+solc_version = "0.8.19"
```

Comment:
let's not downgrade the version of the compiler, only the contracts. Also, would need to investigate the optimisations that we removed because they're applied automatically in the new solidity version, whether we need to re-introduce them.

also, would be good to have more information on why this change is needed